File: data-demo/src/main/java/mwangli/online/datademo/bean/CovidDTO.java
package mwangli.online.datademo.bean;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

import java.util.List;

/**
 * @author mwangli
 * @date 2022/3/28 10:56
 */
@Data
@ToString
@NoArgsConstructor
@AllArgsConstructor
public class CovidDTO {
    private String provinceName;
    private String provinceShortName;
    private Integer currentConfirmedCount;
    private Integer confirmedCount;
    private Integer confirmedIncr;
    private Integer suspectedCount;
    private Integer curedCount;
    private Integer deadCount;
    private String locationId;
    private String pid;
    private String statisticsData;
    private String cities;
    private String cityName;
    private String dateId;
}

File: spark-demo/src/main/scala/mwang/online/bean/CovidDTO.scala
package mwang.online.bean

case class CovidDTO(
   dateId: String,
   provinceShortName: String,
   cityName: String,
   locationId: String,
   pid: String,
   statisticsData: String,
   currentConfirmedCount: Long,
   confirmedCount: Long,
   confirmedIncr: Long,
   suspectedCount: Long,
   curedCount: Long,
   deadCount: Long
)
